TheInternet/HomePage.cs

```diff
@@ -11,9 +11,15 @@ public HomePage(IWebDriver driver): base(driver)
     }
 
     private IWebElement formAuthenticationLink => driver.FindElement(By.CssSelector("a[href='/login']"));
+    private IWebElement hoversLink => driver.FindElement(By.CssSelector("a[href='/hovers']"));
 
     public void clickFormAuthentication()
     {
         clickLink(formAuthenticationLink);
     }
+
+    public void clickHovers()
```

Comment:
Style note:

You may already know this, but just in case....

In JavaScript and Java, generally the only thing that starts with a capital is class names. In C# the convention is to capitalize anything that's public (can be used from outside of the class). Most C#ers like that because it makes it clear at a glance whether the member (anything living inside a class) is private or public.

The compiler never cares how you name things but some tools (e.g. resharper) do nag you to follow these common conventions